go/libkb/passphrase_helper.go

```diff
@@ -171,6 +179,7 @@ func DefaultPassphrasePromptArg(m MetaContext, username string) keybase1.GUIEntr
 // hint otherwise.
 type PassphraseChecker interface {
 	Check(MetaContext, string) error
+	Automutate(MetaContext, string) (string, error)
```

Comment:
What about moving this function into libkb/passphrase_helper.go?  Seems weird for the Checker to be doing anything besides Check.

Reply:
Do you mean something like an `automutate` function being passed to `GetPassphraseUntilCheck`, where it would be nil if it wasn't getting a paperkey?